Patch origin: upstream
Patch status: already part of 2.4 and HEAD (bacport to 2.2 is unclear)

https://svn.apache.org/viewvc?view=revision&revision=731058

--- configure.in	2014-03-28 07:05:06.506452975 -0700
+++ configure.in	2014-03-28 07:08:12.121620041 -0700
@@ -184,7 +184,7 @@
 
 if test "$PCRE_CONFIG" != "false"; then
   AC_MSG_NOTICE([Using external PCRE library from $PCRE_CONFIG])
-  APR_ADDTO(CFLAGS, [`$PCRE_CONFIG --cflags`])
+  APR_ADDTO(PCRE_INCLUDES, [`$PCRE_CONFIG --cflags`])
   APR_ADDTO(LIBS, [`$PCRE_CONFIG --libs`])
 else
   # Build the bundled PCRE
@@ -219,6 +219,9 @@
 APR_ADDTO(INCLUDES, `$apr_config --includes`)
 APR_ADDTO(INCLUDES, `$apu_config --includes`)
 
+dnl Add in path to PCRE includes
+APR_ADDTO(INCLUDES, $PCRE_INCLUDES)
+
 echo $ac_n "${nl}Applying OS-specific hints for httpd ...${nl}"
 
 case $host in
